cmd/start.go

```diff
@@ -1,24 +1,42 @@
 package cmd
 
 import (
+	"fmt"
+	"github.com/NodeFactoryIo/vedran/internal/auth"
 	"github.com/NodeFactoryIo/vedran/internal/router"
 	"github.com/spf13/cobra"
 	"log"
 	"net/http"
 )
 
+var (
+	authSecretFlag string
+)
+
 var startCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Starts vedran load balancer",
 	Run:   startCommand,
 }
 
 func init() {
+	startCmd.Flags().StringVar(
+		&authSecretFlag,
+		"auth-secret",
+		"",
+		"example flag")
 	RootCmd.AddCommand(startCmd)
 }
 
 func startCommand(_ *cobra.Command, _ []string) {
+	err := auth.SetAuthSecret(authSecretFlag)
+	if err != nil {
+		// terminate app: no auth secret provided
+		log.Fatal(fmt.Sprintf("Unable to start vedran load balancer: %v", err))
+	}
+
 	log.Println("Starting vedran load balancer on port :4000...")
-	err := http.ListenAndServe(":4000", router.CreateNewApiRouter())
+
+	err = http.ListenAndServe(":4000", router.CreateNewApiRouter())
 	log.Fatal(err)
```

Comment:
Can we decouple buisness logic from cli.
Like extract all of this into s.StartLoadBalancer(secret, port, etc) method. In cli keep only argument casting or parsing.

Reply:
@mpetrunic for sure, but I think this is out of the scope for this PR. I will be restructuring this code soon when I will be closing issue #7 in the next PR.